generators/entity/templates/src/main/java/package/service/impl/_EntityServiceImpl.java

```diff
@@ -95,8 +95,14 @@ public void delete(<%= pkType %> id) {
      *  @param query the query of the search
      *  @return the list of entities
      */<% if (databaseType == 'sql') { %>
-    @Transactional(readOnly = true) <% } %>
-    public List<<%= instanceType %>> search(String query) {
-        <%- include('../../common/search_template', {viaService: viaService}); -%>
+    @Transactional(readOnly = true)<% } %>
```

Comment:
The idea to use templates was to minimize code duplication coz you will do same code when using  a repo directly in resources or when using it in a service and hence the viaService if else in the template.

Isnt the generated code the same for this method and the resource method if used without a service?  If so cant template be resused by passing via service false here as I was doing earlier?